CODINGSTANDARDS.md

```diff
@@ -219,11 +219,11 @@ should also have a proper signature above it. When you invoke the function, it
 should also have the result of that invocation commented below. 
 
 ```javascript
-// yell :: String -> String
+/** yell :: String -> String */
```

Comment:
Because this is the standards for the documentation, think we are going to want to keep these as they were.